data/moves.js

```diff
@@ -11324,15 +11324,15 @@ exports.BattleMovedex = {
 			onLockMove: 'rollout',
 			onStart: function () {
 				this.effectData.hitCount = 1;
+				this.effectData.trueDuration = 5;
```

Comment:
Maybe use a different word than "true" in front of "duration"? The way the variable is currently named, I don't have a good idea of what exactly it tracks.